Document table name BC break

goetas opened the pull request:
In `doctrine/migrations` 3.x and `doctrine/DoctrineMigrationsBundle` 3.x the default value for  `table name` is `doctrine_migration_versions`.

In the previous 2.x versions the doctrine/migrations library was using as default `doctrine_migration_versions` while the bundle was using `migration_versions`.

Closes https://github.com/doctrine/migrations/issues/1002

greg0ire commented on UPGRADE.md:
```diff
@@ -56,6 +56,17 @@ After
 
+
+- The default for `table_name` changed from `migration_versions` to `doctrine_migration_versions`. If you did not
+specify the `table_name` option, now  need to declare it explicitly to not loose migration data.
```
```suggestion
specify the `table_name` option, you now need to declare it explicitly to not loose migration data.
```

guilliamxavier replied:
also `loose` -> `lose`

goetas commented:
I have also removed the some upgrade instructions that are relative to the `doctrine/migrations` library and added a reference to its dedicated upgrading document.
